### ISSUE TITLE
Can't rename preset name in Game Cooker

### ISSUE THREAD

**Issue by TheJustLink** [FlaxEngine/FlaxEngine#1006]
Can't rename preset name in Game Cooker.

![image](https://user-images.githubusercontent.com/29580493/232316492-ee49e8ab-100f-4d18-9c28-467a9858b4e0.png)

**Flax version:** Last 1.5

**Comment by mafiesto4** [comment]
Yeah this window is pretty old and could be improved for better usability
